hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/ConsumerMetrics.java

```diff
@@ -31,61 +26,54 @@ public  <T> void registerQueueUtilizationGauge(T obj, String queueName, ToDouble
 
     public HermesCounter queueFailuresCounter(String name) {
         return HermesCounters.from(
-                meterRegistry.counter("queue." + name + ".failures"),
-                hermesMetrics.counter("queue." + name + ".failures")
+                meterRegistry.counter("queue." + name + ".failures")
         );
     }
 
     public <T> void registerConsumerProcessesThreadsGauge(T obj, ToDoubleFunction<T> f) {
-        gaugeRegistrar.registerGauge(THREADS, "consumer-processes.threads", obj, f);
+        gaugeRegistrar.registerGauge("consumer-processes.threads", obj, f);
     }
 
     public <T> void registerRunningConsumerProcessesGauge(T obj, ToDoubleFunction<T> f) {
-        hermesMetrics.registerRunningConsumerProcessesCountGauge(() -> (int) f.applyAsDouble(obj));
         meterRegistry.gauge("consumer-processes.running", obj, f);
     }
 
     public <T> void registerDyingConsumerProcessesGauge(T obj, ToDoubleFunction<T> f) {
-        hermesMetrics.registerDyingConsumerProcessesCountGauge(() -> (int) f.applyAsDouble(obj));
         meterRegistry.gauge("consumer-processes.dying", obj, f);
     }
 
     public <T> void registerBatchBufferTotalBytesGauge(T obj, ToDoubleFunction<T> f) {
-        gaugeRegistrar.registerGauge(BATCH_BUFFER_TOTAL_BYTES, "batch-buffer.total-bytes", obj, f);
+        gaugeRegistrar.registerGauge("batch-buffer.total-bytes", obj, f);
```

Comment:
Why did you decide to remove `BATCH_BUFFER_TOTAL_BYTES`? To be consistent with other functions?

Reply:
yes